inference-engine/ie_bridges/c/samples/object_detection_sample_ssd/c_w_dirent.h

```diff
@@ -96,13 +128,18 @@ static DIR *opendir(const char *dirPath) {
     dp->hFind = FindFirstFileA(ws, &dp->FindFileData);
     dp->FindFileData.dwReserved0 = dp->hFind != INVALID_HANDLE_VALUE;
     free(ws);
-    if (isValid(dp)) {
+    if (!isValid(dp)) {
```

Comment:
Could you please fix the functional issues in separate PR? looks like we have no tests for this case.

Reply:
move it